routes/settings.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const router = express.Router();
+const _ = require('lodash')
+const jwtUtils = require('../utils/jwt-utils')
+
+// Import Classes
+const { Settings } = require('../classes/Settings.js')
+
+router.get('/:siteName/settings', async function(req, res, next) {
+  try {
+    const { oauthtoken } = req.cookies
+    let { access_token } = jwtUtils.verifyToken(oauthtoken)
+    const { siteName } = req.params
+
+    const settingsFile = new Settings(access_token, siteName)
```

Comment:
Should it be called `settingsFile`? Since technically, under the hood, we're altering and retrieving content from two files. Maybe just `settings`?

Reply:
done